py/Pipfile.lock

```diff
@@ -5,15 +5,15 @@
         },
         "host-environment-markers": {
             "implementation_name": "cpython",
-            "implementation_version": "0",
+            "implementation_version": "3.6.4",
```

Comment:
Thanks for your work!

Could you give us more details about the code, where is changed by you and where is generated automatically 😄

Reply:
all changed automatically

Reply:
Then LGTM